apps/storefront/src/hooks/dom/useCartToQuote.ts

```diff
@@ -58,6 +59,20 @@ const useCartToQuote = ({ setOpenPage, cartQuoteEnabled }: MutationObserverProps
 
   const { pathname } = window.location;
 
+  useEffect(() => {
+    const showRejectedAccountTip = () => {
+      if (![CHECKOUT_URL].includes(pathname)) return;
+
+      if (companyStatus !== CompanyStatus.REJECTED) return;
```

Comment:
Please, follow standard coding practices from BC. Always include curly braces in `if` statements.